packages/@o3r/testing/src/core/playwright/angular-materials/autocomplete-material.ts

```diff
@@ -16,7 +16,7 @@ export class MatAutocomplete extends O3rElement implements MatAutocompleteProfil
     await this.setValue(value);
     await this.click();
     const element = this.sourceElement.page;
-    const matOptions = element.locator('.mat-option');
+    const matOptions = element.locator('mat-option');
```

Comment:
Shouldn't we put `.mat-option, mat-option` to not break the previous behaviour ?

Reply:
The tag would select both mdc and legacy and imo is more future proof as classes can change with each angular release

Reply:
as we are on a released version

Reply:
done